### PR TITLE
WIP:ghc 8.6.1

### PR DESCRIPTION
**Pull request by mpickering**
This requires an unreleased version of proto-lens and also to manually
put the right proto-lens-protoc on your path.

### CODE REVIEW

**Comment by mpickering** [comment]
`proto-lens` has now been released and supports `ghc-8.6.1` which unblocks this.

**Comment by mpickering** [comment]
The 8.6.1 build now passes.

**Comment by mpickering** [comment]
CI is now green. I haven’t been careful about checking warnings so I will have to check that as well.

File: stack-ghc861.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- haskell-indexer-pathutil
+- haskell-indexer-pipeline-ghckythe
+- haskell-indexer-pipeline-ghckythe-wrapper
+- haskell-indexer-translate
+- kythe-proto
+- kythe-schema
+- text-offset
+
+extra-deps:
+
```
Comment (ivan444) [inline stack-ghc861.yaml]: How come we don't need the extra deps for ghc861? Although, CI is passing for this version. I might be missing something...

Comment (mpickering) [reply stack-ghc861.yaml]: Because it's using the nightly resolver which has the right packages already.

**Pull request merged**